Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2661,15 +2661,23 @@ function Set-CustomDRS {
         [Parameter(Mandatory = $true)]
         [String]$ClustersToChange,
         [Parameter(Mandatory = $true,
-            HelpMessage = "The DRS setting. Default of 3 or more conservative of 4.")]
+            HelpMessage = "The DRS setting. Default of 3 or more conservative of 2 or less conservative 4.")]
         [ValidateRange(3, 4)]
```

Comment:
Does this range validation make sense if 2 and 1 are handled in the switch?

Reply:
Before they could only go one step more conservative. Just letting them know that they have more options.

Reply:
I mean in line 2665

Reply:
Of course you are correct! I completely overlooked that. I've pushed the change.